flow-typed/platform.js

```diff
@@ -16,6 +16,27 @@ declare type MWPState = {
 		enableServiceWorker: boolean,
 		requestLanguage: string,
 		supportedLangs: Array<string>,
+		member:
+			| {|
+					id: number,
+					status: number,
+					timestamp: number,
+					bs: number,
+					// time zone
+					tz: string,
+					zip: number,
+					country: string,
+					city: string,
+					state?: string,
+					lat: number,
+					lon: number,
+					ql: boolean,
+					scope?: string,
+					// whether the member selected remember me or not
+					rem?: number,
+					org?: 1,
```

Comment:
what is org?

Reply:
whether the member is an organizer or not. added a comment.

Reply:
it'll be `org=1` or org won't be provided if not an org